bridgesync/backfill_tx_sender.go

```diff
@@ -163,14 +163,16 @@ func (b *BackfillTxnSender) getRecordsNeedingBackfillCount(ctx context.Context,
 	query := fmt.Sprintf(`
 		SELECT COUNT(*)
 		FROM %s
-		WHERE txn_sender = '' OR txn_sender IS NULL OR from_address = '' OR from_address IS NULL
+		WHERE (txn_sender = '' OR txn_sender IS NULL OR from_address = '' OR from_address IS NULL)
```

Comment:
why are we making changes in backfilling
whenever this feature goes live, it will be ahead of txns already having txn_sender and from_address so i dont think that data will be missing anyway